### PR TITLE
Support for Standard, Wild and set rotations

### PR DESCRIPTION
**Pull request by djkaty**
Cumulatively these commits have the following net effect:

+ cards from adventures now have crafting costs assigned
+ standard mode intro can now be triggered
+ hang during standard mode intro fixed
+ hang during choosing deck against innkeeper fixed
+ can now switch between wild and standard in crafting mode
+ can now switch between wild and standard in play mode
+ cards from adventures can now be crafted
+ wild cards can once again be crafted
+ set names coming in wrong places on collection filter menu fixed
+ you can now queue against the innkeeper (instant crash)
+ you can now convert a deck between wild and standard (not persistent yet)
+ client will no longer believe wild cards can be used in standard
+ Whispers of The Old Gods cards can now be crafted
+ the standard tutorial and set rotation intro will be skipped by default
+ wild cards can no longer be added to standard decks
+ the Feast of Winterveil is now over. Happy Standard format!

### CODE REVIEW

File: pegasus/db.go [hunk 1]
```diff
@@ -265,3 +266,9 @@ type CollectionCard struct {
 	Premium   int32
 	Num       int32
 }
+
+type SpecialEventTiming struct {
+	Event     string
+	Start     uint64
+	End       uint64
+}
```
Comment (jleclanche) [inline pegasus/db.go]: No newline at EOF (please set your editor to always ensure there is one, and you won't have that problem again).

File: scripts/initialize_database.py [hunk 2]
```diff
@@ -246,6 +246,53 @@ def main():
 			cards.append((id, deck_id, card_id, premium, 2))
 		connection.executemany("INSERT INTO deck_card VALUES (?, ?, ?, ?, ?)", cards)
 
+	# sepcial events
```
Comment (jleclanche) [inline scripts/initialize_database.py]: typo :)

File: scripts/initialize_database.py [hunk 3]
```diff
@@ -246,6 +246,53 @@ def main():
...
+	# we need this otherwise crafting of rotated sets fails
+	special_events = [
```
Comment (jleclanche) [inline scripts/initialize_database.py]: Where's this list from?

File: scripts/initialize_database.py [hunk 4]
```diff
@@ -281,10 +281,18 @@ def main():
 		"set_rotation_2016_questline",
 	]
 
+	def event_ends(event):
```
Comment (jleclanche) [inline scripts/initialize_database.py]: That's a nasty way to do it don't you think?

**Comment by djkaty** [comment]
I've pushed one additional commit (fdd75fb) which fixes a bug where a deck's standard/wild setting wasn't persisted to the database when 'convert to standard' or 'convert to wild' was used in the collection manager.

I consider this to be the end of this branch, afaik all deck-related standard/wild and card-related set rotation code is feature complete.

**Comment by jleclanche** [comment]
gtm

**Pull request merged**